src/js/containers/covid19/budgetCategories/BudgetCategoriesTableContainer.jsx

```diff
@@ -195,9 +195,6 @@ const BudgetCategoriesTableContainer = (props) => {
         else if (props.type === 'object_class' && spendingCategory === 'total_spending') {
             unlinkedName = 'Unknown Object Class (Unlinked Data)';
         }
-        else if (spendingCategory === 'award_spending') {
-            unlinkedName = 'Number of Unlinked Awards';
-        }
 
```

Comment:
if we're removing the unlinked row, can we just delete this entire fn `addUnlinkedData`?

Reply:
Also we should delete the callers of this method, correct?

Reply:
@MaxwellKendall I opened a new PR here (since I accidentally merged dev instead of qat into this one) #2231 
But to answer your question, `addUnlinkedData` is still used for the "Total Spending" view Unlinked data row.